Tests/AnyLintTests/StatisticsTests.swift

```diff
@@ -7,91 +7,105 @@
 
         let checkInfo3 = CheckInfo(id: "id3", hint: "hint3", severity: .error)
-        Statistics.shared.found(
-            violations: [Violation(checkInfo: checkInfo3), Violation(checkInfo: checkInfo3), Violation(checkInfo: checkInfo3)],
-            in: CheckInfo(id: "id3", hint: "hint3", severity: .error)
+        Statistics.default.found(
+            violations: [
+                CheckInfo(id: "id3", hint: "hint3", severity: .error):
+                [Violation(checkInfo: checkInfo3), Violation(checkInfo: checkInfo3), Violation(checkInfo: checkInfo3)]
+            ]
         )
 
-        XCTAssertEqual(Statistics.shared.executedChecks, [checkInfo1, checkInfo2, checkInfo3])
-        XCTAssertEqual(Statistics.shared.violationsBySeverity[.info]!.count, 1)
-        XCTAssertEqual(Statistics.shared.violationsBySeverity[.warning]!.count, 2)
-        XCTAssertEqual(Statistics.shared.violationsBySeverity[.error]!.count, 3)
-        XCTAssertEqual(Statistics.shared.violationsPerCheck.keys.count, 3)
+        XCTAssertEqual(Statistics.default.executedChecks, [checkInfo1, checkInfo2, checkInfo3])
+        XCTAssertEqual(Statistics.default.violationsBySeverity[.info]!.count, 1)
+        XCTAssertEqual(Statistics.default.violationsBySeverity[.warning]!.count, 2)
+        XCTAssertEqual(Statistics.default.violationsBySeverity[.error]!.count, 3)
+        XCTAssertEqual(Statistics.default.violationsPerCheck.keys.count, 3)
     }
 
     func testLogSummary() { // swiftlint:disable:this function_body_length
-        Statistics.shared.logCheckSummary()
+        Statistics.default.logCheckSummary()
         XCTAssertEqual(TestHelper.shared.consoleOutputs.count, 1)
         XCTAssertEqual(TestHelper.shared.consoleOutputs[0].level, .warning)
         XCTAssertEqual(TestHelper.shared.consoleOutputs[0].message, "No checks found to perform.")
 
         TestHelper.shared.reset()
 
         let checkInfo1 = CheckInfo(id: "id1", hint: "hint1", severity: .info)
-        Statistics.shared.found(
-            violations: [Violation(checkInfo: checkInfo1)],
-            in: checkInfo1
-        )
+        Statistics.default.found(violations: [checkInfo1: [Violation(checkInfo: checkInfo1)]])
 
         let checkInfo2 = CheckInfo(id: "id2", hint: "hint2", severity: .warning)
-        Statistics.shared.found(
+        Statistics.default.found(
             violations: [
-                Violation(checkInfo: checkInfo2, filePath: "Hogwarts/Harry.swift"),
-                Violation(checkInfo: checkInfo2, filePath: "Hogwarts/Albus.swift"),
-            ],
-            in: CheckInfo(id: "id2", hint: "hint2", severity: .warning)
+                CheckInfo(id: "id2", hint: "hint2", severity: .warning):
+                    [
+                        Violation(checkInfo: checkInfo2, filePath: "Hogwarts/Harry.swift"),
+                        Violation(checkInfo: checkInfo2, filePath: "Hogwarts/Albus.swift"),
+                    ]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Multi-line collection literals should have trailing commas.](https://app.codacy.com/gh/Flinesoft/AnyLint/pullRequest?prid=5691372)